Add few minor details to Pong tutorial (#1079)
amethyst/amethyst: book/src/pong-tutorial/pong-tutorial-02.md

# }
```

> If you would try to run the game right now, you'd see a panic message...

Author: Sure, I removed it

> Even if the game would start successfully, it is still would be only black screen.

Reviewer: ```suggestion
Even if the game would start successfully, you would only be seeing a black screen.
```
